src/core/socket.c

```diff
@@ -82,7 +82,7 @@ static void socket_init(Unit *u) {
         assert(u);
         assert(u->load_state == UNIT_STUB);
 
-        s->backlog = SOMAXCONN;
+        s->backlog = -1;
```

Comment:
Please change the type of `Socket.backlog` from `unsigned` to `int`.

Also, please change the conf parser to `config_parse_int()`. See src/core/load-fragment-gperf.gperf

Reply:
huh.. yeah thanks . will do.